src/login/oidc/redirectHandler/InactionRedirectHandler.ts

```diff
@@ -27,13 +27,17 @@ import { ISessionInfoManager } from "../../../sessionInfo/SessionInfoManager";
 @injectable()
 export default class InactionRedirectHandler implements IRedirectHandler {
   constructor(
+    // FIXME: I encountered a number of ESLint issues that had been missed due to an invalid setup,
+    //        and I'm unsure whether TSyringe depends on a specific variable name.
```

Comment:
`TSyringe` should only depend on the `@inject("sessionInfoManager")` notation, but when splitting up the codebase I'll clean up these unsused dependencies.

Reply:
I don't know why I can't respond to your other comments, but:

- Prettier on tests: 0fb275af6f2d8a8b381d2889d5212767143e9324
- Audit of examples (barely any noise): b6a94d0a437e67359a1e077defb61a4c92cf9b55
- Unused injected dependency: 76283dcacf82bd7e5722580a2d27aece9b7d3163